tests/test_load_data.py

```diff
@@ -215,3 +215,61 @@ def test_api_fluid_idefix(test_data_dir):
             fluid="dust1",
             directory=test_data_dir / "idefix_spherical_planet3d",
         )
+
+
+def test_api_vtk_slices_idefix(test_data_dir):
+    on = 9
+    ds = GasDataSet(test_data_dir / "idefix_vtk_slices" / f"data.{on:04d}.vtk")
+
+    ds_phi_cut = GasDataSet(
+        test_data_dir / "idefix_vtk_slices" / f"slice1.{on:04d}.vtk"
+    )
+    np.testing.assert_array_almost_equal(
+        ds["RHO"].azimuthal_at_phi(np.pi).data, ds_phi_cut["RHO"].data
+    )
+    np.testing.assert_array_almost_equal(
+        ds["RHO"].azimuthal_at_phi(np.pi).coords.r, ds_phi_cut.coords.r
+    )
+    np.testing.assert_array_almost_equal(
+        ds["RHO"].azimuthal_at_phi(np.pi).coords.theta, ds_phi_cut.coords.theta
+    )
```

Comment:
Suggestion to improve readability
```suggestion
    rho_slice_pp = ds["RHO"].azimuthal_at_phi(np.pi)
    rho_slice_cut = ds_phi_cut["RHO"]
    np.testing.assert_array_almost_equal(rho_slice_pp.data, rho_slice.data)
    
    for coord in ("r", "theta"):
    		np.testing.assert_array_almost_equal(
    		    getattr(rho_slice_pp.coords, coord),
    		    getattr(rho_slice_cut.coords, coord),
    		)
    )
```
(The rest of the test should be refactored as a `@pytest.mark.parametrize`d test) 

It would also be great to add an explicit tolerance.

Reply:
OK I can rewrite all the test this way, but not sure to know how to use  `@pytest.mark.parametrize` when the parameter to change is a method ? :thinking: